### DISCUSSION TITLE
how could this library be used with React Hook Form?

### DISCUSSION THREAD

**Discussion by lucksp** [data-driven-forms/react-forms#1295] [General]
In reading the docs, the `usage` section explains the requirement to use your FormProvider: https://github.com/data-driven-forms/react-forms#usage

We always use React Hook Form...can this library be used with it, to access the schema mapping & logic to render?

**Comment by rvsia** [comment]
Hello @lucksp,

Data Driven Forms uses React Final Form as its state management library and it cannot be changed to a different library, however you should not need to change it as DDF already provides all the features.
